integration/matchers/common.go

```diff
@@ -0,0 +1,81 @@
+package matchers
+
+import (
+	"archive/tar"
+	"errors"
+	"fmt"
+	"io"
+	"regexp"
+	"strings"
+
+	v1 "github.com/google/go-containerregistry/pkg/v1"
+)
+
+func match(expected, actual interface{}, matcher func(*tar.Header, io.Reader) (bool, error)) (bool, error) {
```

Comment:
nit pick, but if these are top-level jam matchers, I think the names should indicate that this is an image-specific `match` function.

Reply:
I've changed this to `matchImage`. Does that work?